bin/rucio

```diff
@@ -212,23 +212,26 @@ def exception_handler(function):
                     pass
                 logger.error('Specified account %s does not have an associated identity.' % used_account)
             else:
-                logger.error(error)
                 logger.debug(traceback.format_exc())
+                contact= config_get('policy', 'support', raise_exception=False)
+                support = ('Please followup with all relevant information at: ' + contact) if contact else ''
                 logger.error('\nThe object is missing this property: %s\n'
                              'This should never happen. Please rerun the last command with the "-v" option to gather more information.\n'
-                             'Please followup with all relevant information at: %s' % (str(error), config_get('policy', 'support')))
```

Comment:
One possibility is to define a default value instead like  `config_get('policy', 'support', 'github.com/rucio/issues')`

Reply:
Yeah I know but what to write in there?

Reply:
`https://github.com/rucio/rucio/issues`

:-D

Reply:
What I wanted to say is then we could hardcode it anyway ^^